backend/controllers/patient-controller.js

```diff
@@ -29,19 +32,87 @@ function getRequiredDetails(req, res) {
 function updateRequiredDetails(req, res) {
     RequiredDetails.update(
         {
-            WeightRequired: req.body[0].Weight,
-            TemperatureRequired: req.body[1].Temperature,
-            SymptomsRequired: req.body[2].Symptoms,
+            WeightRequired: req.body.weight ? "1" : "0",
+            TemperatureRequired: req.body.temperature ? "1" : "0",
+            SymptomsRequired: req.body.symptoms ? "1" : "0",
         },
         {
-            where: { Patient_PatientId: req.params.patientId },
+            where: { Patient_PatientId: req.body.patientId },
         }
     )
-        .then(res.status(200).json({ message: "Details have been updated successfully" }))
-        .catch(() => res.status(400).json({ message: "Could not update details." }));
+        .then(res.status(200).json({ message: "Successfully updated details." }))
+        .catch(err => {
+            console.log(err);
```

Comment:
do we wanna remove this or nah?

Reply:
Uhhhh we could but is there a reason to?